kinyarwanda-net/mobile-app
# KN-#699bI3vZ: setup  project

delivers(chore): #699bI3vZ

#### What does this PR do?
- Install require packages and libraries
- Organize folder structure
#### Description of Task to be completed?
- Test environment
- Folder structure
#### How should this be manually tested?
- Clone the repo locally 
- once cloned go to cloned repo
-  In terminal type ```npm i ``` to install required packages
- Run ```npm start  ``` or ```expo start``` to start expo metro bundler
- Run ```i``` to start the app in ```ios``` emulator or ```a``` from android
- Or scan the provided QRCode in metro bundler to run the app in physical devices``` you should have expo installed in your device``` 
- Start using  the app
#### Any background context you want to provide?
- N/A
#### What are the relevant Trello stories?
- [#699bI3vZ](https://trello.com/c/699bI3vZ/4-setup-a-react-native-app)
#### Screenshots (if appropriate)
- NA
#### Questions:
- N/A
